cmd/system-shutdown/system-shutdown.c

```diff
@@ -93,7 +101,19 @@ int main(int argc, char *argv[])
 		}
 	}
 
-	reboot(cmd);
+	// glibc reboot wrapper does not expose the optional reboot syscall
+	// parameter
```

Comment:
Not sure what you mean, /usr/include/linux/reboot.h only shows some macros, and does not have any include directive.